msg-socket/src/pub/driver.rs

```diff
@@ -51,12 +45,7 @@ impl<T: ServerTransport> Future for PubDriver<T> {
                         // Run custom authenticator
                         debug!("Authentication passed for {:?} ({})", auth.id, auth.addr);
 
-                        let mut framed = Framed::with_capacity(
-                            auth.stream,
-                            pubsub::Codec::new(),
-                            this.options.backpressure_boundary,
-                        );
-
+                        let mut framed = Framed::new(auth.stream, pubsub::Codec::new());
                         framed.set_backpressure_boundary(this.options.backpressure_boundary);
```

Comment:
Because the publisher does not have a very high read volume, so the default (8192 bytes) should be good enough. The only messages it reads are control messages related to subscribing and unsubscribing.